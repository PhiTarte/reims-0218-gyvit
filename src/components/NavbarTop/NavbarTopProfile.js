//TODO: create component Profile

import React from "react"
import { Media } from "reactstrap"
import PropTypes from "prop-types"
import "bootstrap/dist/css/bootstrap.min.css"
import "./styles-navbar-top/navbarProfile.css"

const NavbarTopProfile = ({ firstname, lastname, business }) => (
  <div>
    <Media>
      <Media href="#">
        <Media
          object
          className="pic-profile rounded-circle"
          src="img/kevinMarlot.jpeg"
          alt={firstname}
        />
      </Media>
      <Media body>
        <Media className="id-profile" heading>
          <span>{firstname}</span> <span>{lastname}</span>
        </Media>

        <Media className="business-profile">{business}</Media>
      </Media>
    </Media>
  </div>
)

NavbarTopProfile.propTypes = {
  lastname: PropTypes.string.isRequired,
  firstname: PropTypes.string.isRequired,
  business: PropTypes.string.isRequired
}

export default NavbarTopProfile
