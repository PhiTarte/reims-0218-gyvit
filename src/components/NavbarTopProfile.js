//TODO: create component Profile

import React from "react"
import { NavItem, Media } from "reactstrap"
import "bootstrap/dist/css/bootstrap.min.css"

const NavbarTopProfile = () => (
  <NavItem>
    <Media>
      <Media rounded-circle="true" href="#">
        <Media
          object
          src="http://via.placeholder.com/64x64"
          alt="Kevin Marlot"
        />
      </Media>
      <Media body>
        <Media heading>Kevin Marlot</Media>
        Meduza
      </Media>
    </Media>
  </NavItem>
)

export default NavbarTopProfile
