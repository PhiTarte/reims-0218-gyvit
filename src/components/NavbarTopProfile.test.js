//TODO: test : component Profile

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import NavbarTopProfile from "./NavbarTopProfile"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<NavbarTopProfile />, div)
  ReactDOM.unmountComponentAtNode(div)
})

const profile = { firstname: "", lastname: "", business: "" }

it("renders correctly", () => {
  const tree = renderer.create(<NavbarTopProfile />).toJSON()
  expect(tree).toMatchSnapshot()
})
