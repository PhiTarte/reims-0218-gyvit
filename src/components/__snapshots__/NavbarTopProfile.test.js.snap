// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="col-3 nav-item"
>
  <div
    className="media"
  >
    <div
      className="media"
      href="#"
      rounded-circle="true"
    >
      <img
        alt="kevin"
        className="media-object"
        src="http://via.placeholder.com/64x64"
      />
    </div>
    <div
      className="media-body"
    >
      <h4
        className="media-heading"
      >
        <span>
          kevin
        </span>
        Â 
        <span>
          Marlot
        </span>
      </h4>
      Meduza
    </div>
  </div>
</li>
`;
